examples/builder/vsphere-iso/alpine/alpine-3.20.3.pkr.hcl

```diff
@@ -12,20 +21,21 @@ source "vsphere-iso" "autogenerated_1" {
   CPUs                 = 1
   RAM                  = 512
   RAM_reserve_all      = true
-  boot_command         = ["root<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "setup-alpine -f /media/floppy/answerfile<enter>", "<wait5>", "jetbrains<enter>", "jetbrains<enter>", "<wait5>", "y<enter>", "<wait10><wait10><wait10><wait10>", "reboot<enter>", "<wait10><wait10>", "root<enter>", "jetbrains<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "/media/floppy/SETUP.SH<enter>"]
+  boot_command         = ["root<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "setup-alpine -f /media/floppy/answerfile<enter>", "<wait5>", "VMw@re1!<enter>", "VMw@re1!<enter>", "<wait5>", "y<enter>", "<wait10><wait10><wait10><wait10><wait10><wait10>", "reboot<enter>", "<wait10><wait10>", "root<enter>", "VMw@re1!<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "/media/floppy/SETUP.SH<enter>"]
   boot_wait            = "15s"
   disk_controller_type = ["pvscsi"]
   floppy_files         = ["${path.root}/answerfile", "${path.root}/setup.sh"]
   guest_os_type        = "other3xLinux64Guest"
   host                 = "esxi-01.example.com"
   insecure_connection  = true
-  iso_paths            = ["[datastore1] ISO/alpine-standard-3.8.2-x86_64.iso"]
+  iso_paths            = ["[datastore1] ISO/alpine-standard-3.20.3-x86_64.iso"]
   network_adapters {
     network_card = "vmxnet3"
+    network      = "example-workload"
   }
   password     = "VMw@re1!"
```

Comment:
```suggestion
```

I don't think this is needed.

Reply:
I can comment it out, but in some cases like mine it is required.

```
Build 'vsphere-iso.autogenerated_1' errored after 117 milliseconds 73 microseconds: error creating virtual machine: more than one network found on host [hostname], specify the network
```

Reply:
That's the password for the vSphere admin server yeah? I think it has to be there.